examples/lib/stories/input/draggables.dart

```diff
@@ -58,13 +62,15 @@ class DraggableSquare extends PositionComponent
 
 class DraggablesGame extends BaseGame with HasDraggableComponents {
   final double zoom;
+  late final DraggableSquare square;
 
   DraggablesGame({required this.zoom});
 
   @override
   Future<void> onLoad() async {
     camera.zoom = zoom;
-    add(DraggableSquare());
+    square = DraggableSquare();
+    add(square);
```

Comment:
nit: `add(square = DraggableSquare());` :P (just bc we do that most of the times)